core/templates/dev/head/components/ratingsSpec.js

```diff
@@ -116,14 +116,6 @@ describe('Ratings from frequencies directive', function() {
       '4': 0,
       '5': 0
     })).toBe(undefined);
-
-    expect(ctrlScope.computeAverageRating({
```

Comment:
Would you mind leaving a version of this test with a note that it should be updated when the minimum acceptable number is changed back.